PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterCharacterScripts/Sound.rbxmx

```diff
@@ -83,7 +85,15 @@ local Sounds = {}
 do
 	local Figure = script.Parent.Parent
 	Head = Figure:WaitForChild("Head")
-	Humanoid = Figure:WaitForChild("Humanoid")
```

Comment:
also does this mean we need a better way to wait for child, with additional parameters or something?